src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -477,9 +478,14 @@ public static ViewportEvent.ComputeCameraAngles onCameraSetup(GameRenderer rende
         return event;
     }
 
+    public static void onModifyBakingResult(Map<ResourceLocation, BakedModel> models, ModelBakery modelBakery)
+    {
+        ModLoader.get().postEvent(new ModelEvent.ModifyBakingResult(models, modelBakery));
+    }
+
     public static void onModelBake(ModelManager modelManager, Map<ResourceLocation, BakedModel> models, ModelBakery modelBakery)
     {
-        ModLoader.get().postEvent(new ModelEvent.BakingCompleted(modelManager, models, modelBakery));
+        ModLoader.get().postEvent(new ModelEvent.BakingCompleted(modelManager, Collections.unmodifiableMap(models), modelBakery));
```

Comment:
Please modify the javadocs of `ModelEvent.BakingCompleted` to reflect the map's new status as unmodifiable, as well as directing users who wish to modify the map to the new event.